### ISSUE TITLE
shape mismatch 

### ISSUE THREAD

**Issue by nmahammad** [zhijiejia/ChangeNet-Pytorch#6]
Hello, thanks for your work.

The gt labels are in the shape of [3, 224, 224] however the model output is [1, 11, 224, 224] . Here let's say we added batch size 1 to the label. After, it will be  [1, 3, 224, 224] . Here 3 is number of channels however in the output 11 is number of classes. Also since these 2 values mismatch, it is not possible to use loss function. How did you handle this ? Or is there something I'm missing ? Thanks in advance.
